cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-indexes-intro.asciidoc

```diff
@@ -102,24 +119,33 @@ Now we run the same queries again:
 MATCH (n:User) WHERE n.name CONTAINS 'ndy' RETURN n.name;
 ----
 
-This query will operate exactly as before, returning the same single result, because nothing in this query relates to the denylisted property.
+This query will operate exactly as before, returning the same two results, because nothing in this query relates to the blacklisted property.
 
-But consider the full-text index query:
+However, for the query targeting the composite index things have changed.
+
+[source, cypher]
+----
+MATCH (n:User) WHERE n.name CONTAINS 'ndy' AND n.surname IS NOT NULL RETURN n.name;
+----
+
+Since the `surname` property is blacklisted it will appear to always be `null` and the composite index empty. Therefore, query returns no result.
+
+Now consider the full-text index query:
 
 [source, cypher]
 ----
 CALL db.index.fulltext.queryNodes("userNames", "ndy") YIELD node, score
 RETURN node.name
 ----
 
-The problem now is that we do not know if the results provided by the index were because of a match to the `name` or the `surname` property.
-Results from the surname need to be excluded by the security rules, because they require that the user cannot see any `surname` properties.
+The problem that we do not know if the results provided by the index were because of a match to the `name` or the `surname` property remains.
```

Comment:
```suggestion
The problem remains, we do not know if the results provided by the index were because of a match on the `name` or the `surname` property.
```